scss/abstracts/_mixins.scss

```diff
@@ -0,0 +1,40 @@
+@use 'variables' as *;
+
+@mixin flexbox($flex-direction: row, $justify: center, $align: center) {
+    display: flex;
+    flex-direction: $flex-direction;
+    justify-content: $justify;
+    align-items: $align;
+}
+
+@mixin btn-primary($border-radius: 0.3rem) {
```

Comment:
Interesting! I would rather create BEM Block for btn / btn-primmary, but that's also ok!

:+1: :thinking:

Reply:
Thanks, there are three buttons that are styled exactly the same - except the `border-radius` - so I thought it might not be a bad solution to create a mixin for that